configs/.sass-lint.yml

```diff
@@ -40,10 +40,7 @@ rules:
   force-pseudo-nesting: 0
 
   # Name Formats
-  class-name-format:
-    - 1
-    -
-      convention: hyphenatedbem
+  class-name-format: 0
```

Comment:
We are currently using BEM with single and double hyphens. We need to agree the standard then we can put this back in